src/pages/CreatorCenter/components/ClassRegist/components/ClassInfo.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import styled from 'styled-components';
+import InputForm from './InputForm';
+
+function ClassInfo({ title }) {
+  return (
+    <InfoContainer>
+      <InfoTitle>{title}</InfoTitle>
+      <InputForm
+        type="file"
+        subject="커버 이미지"
+        description="커버로 사용할 이미지를 추가해주세요"
+      />
```

Comment:
```suggestion
      {FORM_TEXT_LIST.map(({ id, type, subject, description, borderUnUse }) => {
        return (
          <InputForm
            key={id}
            type={type}
            subject={subject}
            borderUnUse={borderUnUse}
            description={description}
          />
        );
      })}
```
똑같은 inputForm 컴포넌트를 사용하는데, props로 받는 값만 달라진다면
상수데이터를 활용해서 map 메서드를 적용하는 방식으로 구현해주세요!

FORM_TEXT_LIST 가 상수데이터의 이름이 되겠죠?